src/browser/shim.js

```diff
@@ -112,7 +112,7 @@ Shim.prototype.loadFull = function(window, document, immediate, options, callbac
   parentNode.insertBefore(s, f);
 };
 
-Shim.prototype.wrap = function(f, context) {
+Shim.prototype.wrap = function(f, context, _before) {
```

Comment:
Looks like Shim.prototype.wrap and Rollbar.prototype.wrap are almost identical. Can we consolidate?